test/html/pipeline/mention_filter_test.rb

```diff
@@ -72,6 +72,20 @@ def test_links_to_more_info_when_info_url_given
       filter(body, '/', 'https://github.com/blog/821').to_html
   end
 
+  def test_base_url_tilde_with_slash
+    body = "<p>Hi, @jch!</p>"
+    link = "<a href=\"~jch\" class=\"user-mention\">@jch</a>"
```

Comment:
I've reread the tests, and I'm wondering why the `href` doesn't include a `/`.  

If the HTML document is `/blog/posts/jan-3.html` and the link is `<a href="~jch" class="user-mention">@jch</a>`, wouldn't the browser go to  `/blog/posts/~jch` and not `/~jch`?   Am I missing something?  Thanks!